### ISSUE TITLE
Add official archives to list of archives?

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#1902]
`cocopp.archiving.ListOfArchives.lists()` does currently not contain the official archives `cocopp.archiving.official_archives`, should it?

**Comment by brockho** [comment]
That would be, indeed, nice to have.

At the same time, I was wondering why we have `cocopp.archiving` and `cocopp.archives` in parallel. Initially, I had problems to identify in which of the two certain functions are located. Is there a reason to keep the two separate or is it better to merge them (preferably into `archives`)?

**Comment by nikohansen** [comment]
`archiving` is the module that provides classes and methods to manage archives (create, get,...). `archives` is a class instance that contains the "official" archives and is aliased to `archiving.official_archives` (with the possible idea to remove`archives`, but I am not fully convinced that this is a good idea.

**Comment by brockho** [comment]
It seems that this issue can be closed?
```python
In [7]: cocopp.archiving.ListOfArchives.lists()
Out[7]:
{'known_archives': ['https://cma-es.github.io/lq-cma/data-archives/lq-gecco2019',
  'https://numbbo.github.io/cocopp-playground/bbob-constrained/testdata/bbob-constrained'],
 'local_archives': ['C:\\Users\\dimo\\Desktop\\data-archive\\data-archive',
  'C:\\Users\\dimo\\Desktop\\data-archive\\data-archive\\bbob',
  'C:\\Users\\dimo\\Desktop\\data-archive\\data-archive\\bbob-biobj',
  'C:\\Users\\dimo\\Desktop\\data-archive\\data-archive\\bbob-constrained',
  'C:\\Users\\dimo\\Desktop\\data-archive\\data-archive\\bbob-constrained\\2022',
  'C:\\Users\\dimo\\Desktop\\data-archive\\data-archive\\bbob-largescale',
  'C:\\Users\\dimo\\Desktop\\data-archive\\data-archive\\bbob-mixint',
  'C:\\Users\\dimo\\Desktop\\data-archive\\data-archive\\bbob-noisy',
  'C:\\Users\\dimo\\Desktop\\data-archive\\data-archive\\sbox-cost']}
```

**Comment by brockho** [comment]
After discussing with @nikohansen and @ttusar, we realized that these `lists()` are only the ones that I already used.
